ttcli/utils.py

```diff
@@ -220,3 +226,164 @@ def get_confirmation(prompt: str, default: bool = True) -> bool:
             return True
         if answer[0] == "n":
             return False
+
+def is_third_friday(date, tz):
```

Comment:
This could be way simpler, we already have a function `get_third_friday`, so just check if `get_third_friday(date) == date`